vb-be/src/app.module.ts

```diff
@@ -26,4 +35,15 @@ import { RoomModule } from '@modules/room/room.module';
   providers: [AppService],
   exports: [RedisModule, MongoDBModule],
 })
-export class AppModule {}
+export class AppModule implements NestModule {
+  configure(consumer: MiddlewareConsumer) {
+    consumer
+      .apply(ValidateAccessTokenMiddleware)
+      .exclude('/api/auth/authorize', '/api/auth/login', '/api/auth/refresh')
+      .forRoutes({ path: '/api', method: RequestMethod.GET });
```

Comment:
Creating a room is a `POST` method, could you change this middleware to be included for other methods as well?

Reply:
this is resolved right?

Reply:
Not sure, I didn't make any changes.  In my local, it's just being triggered through the `room.module`